pyzx/simulated_annealing.py

```diff
@@ -0,0 +1,60 @@
+from tqdm import tqdm
```

Comment:
Missing the standard header on top of the file. Also should have a top level documentation comment. Missing documentation strings for all the functions.